lib/components/ClinicUserSelect.js

```diff
@@ -41,6 +42,10 @@ var ClinicUserSelect = React.createClass({
     }
   },
 
+  handleOnChange: function(userId) {
+    this.props.setTargetUser(userId, {eventName: metrics.CLINIC_SEARCH_SELECTED});
+  },
```

Comment:
Chatted with @jh-bate a bit about this, and the two of us are agreed that perhaps the next uploader task that touches any React component should include setting up React component testing and writing some tests for that component (exactly what these tests look like to be decided in future discussion) as part of the task. Then as we touch components in the uploader we can try to gradually add tests. Thoughts @krystophv? Sound workable to you?